### PR TITLE
User can Upload Image

### PR DESCRIPTION
**Pull request by tochman**
PT: https://www.pivotaltracker.com/story/show/169667235

```
As a User
In order to show recipe image
I would like to be able to upload an image
```

Skärmavbild 2019-11-13 kl  09 52 19

### CODE REVIEW

File: src/modules/requestRecipes.js [hunk 1]
```diff
@@ -2,28 +2,44 @@ import axios from 'axios';
...
   try {
+    let encodedImage, recipeParams
+    recipeParams = {
+      title: title,
+      ingredients: ingredients,
+      directions: directions
+    }
+
+    if (image) {
+      // If there is an image, we add it to the params we will be sending off
```
Comment (clalil) [inline src/modules/requestRecipes.js]: Please remove comment before merge :)

**Review by beccaburns — APPROVED** [review]

File: cypress/integration/userCanCreateRecipe.spec.js [hunk 2]
```diff
@@ -35,7 +41,7 @@ describe('Creates a recipe', () => {
       response: '{ "error_message": "Unable to create recipe." }',
```
Comment (clalil) [inline cypress/integration/userCanCreateRecipe.spec.js:40]: In the module requestRecipes.js, you are specifying two error responses on line 42. 
'error: error.response.data.error_message || error.message'
Is there a test for the second error message (for not attaching an image) or does it not need to be tested?

File: src/modules/requestRecipes.js (hunk 1 above)
Comment (tochman) [reply src/modules/requestRecipes.js]: Done

File: cypress/integration/userCanCreateRecipe.spec.js (hunk 2 above)
Comment (tochman) [reply cypress/integration/userCanCreateRecipe.spec.js:40]: It is a fallback if the image is bad. No need to test this.

**Review by yasmineezequiel — APPROVED** [review]

**Review by shsteindorsson — APPROVED** [review]

**Pull request merged**